crates/sui-types/src/effects/mod.rs

```diff
@@ -390,7 +390,7 @@ pub trait TransactionEffectsAPI {
     fn unsafe_add_object_tombstone_for_testing(&mut self, obj_ref: ObjectRef);
 }
 
-#[derive(Clone)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct ObjectChange {
```

Comment:
This PR is making the struct `ObjectChange` a new public data structure since its included in the merkle tree now. We should make sure we're happy with doing this. Right now this type is only an internal library type.

Reply:
Removed it after the last conversation with Mark where we decided to commit to less things.